src/Uno.UI.Tests/Windows_UI_Xaml_Data/xBindTests/Given_xBind_Binding.cs

```diff
@@ -1088,8 +1088,8 @@ public async Task When_xLoad_StaticResource()
 
 			SUT.IsTestGridLoaded = false;
 
-			AssertIsNullAsync(() => SUT.TestGrid);
```

Comment:
This assert is now failing after adding the missing `await` :)

Reply:
@jeromelaban Could you take a look at this one please?

Reply:
Note: This doesn't seem like the only test failure related to missing `await`s

Reply:
This looks like a legitimate failure, but I don't know what it is. Let's ignore it with a new issue, I'll take a look, it's not supposed to happen.